frontend/src/views/App/App.tsx

```diff
@@ -1,17 +1,11 @@
 import React from 'react';
+import 'antd/dist/antd.css';
```

Comment:
Importowanie w tym miejscu wydaje się intuicyjne, ale jest trochę problematyczne w przypadku tworzenia testów albo wykorzystywania gdzieś samych komponentów, bez App.tsx.
Zamiast tego możemy zaimportować to w pliku, który jest używany zarówno w testach jaki i podczas odpalania aplikacji. Znasz takie miejsce w aktualnym projekcie?

Reply:
jeżeli chodzi o app providers to zmieniłem ścieżkę, a jak nie to mi na discordzie dopowiesz :D